packages/plugin-defender/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "dist",
+    "rootDir": "src",
+    "resolveJsonModule": true,
+  },
+  "include": [
+    "src/**/*"
+  ],
+  "references": [
+    { "path": "../core" }
```

I think we need this because it's a dependency too.

```suggestion
    { "path": "../core" },
    { "path": "../plugin-hardhat" }
```